packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -64,6 +64,7 @@ class ExpansionTile extends StatefulWidget {
     this.childrenPadding,
     this.backgroundColor,
     this.collapsedBackgroundColor,
```

Comment:
I wonder if these might be better served as MaterialStateProperties, so you can customize the hover color (Or rather, not affect the hover color at all). That might be more discoverable than the blend mode.